tests/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove this file as its usage is discouraged with `py.test`. 

> **avoid “__init__.py” files in your test directories.** This way your tests can run easily against an installed version of mypkg, independently from the installed package if it contains the tests or not.

*source https://pytest.org/latest/goodpractises.html*